act-rules-format.bs

```diff
@@ -512,13 +512,33 @@ Definitions {#definitions}
     </ul>
   </dd>
 
+  <dt><dfn>meet expectations</dfn></dt>
+  <dd>
+    <p>A [=test target=] meets an expectation, when that expectation is true for the test target individually, or when the test target increases a count that is required for the expectation to be true.</p>
+    <div class=example>
+      <p>**Example:** An HTML `img` element an `alt` attribute meet the following expectation:</p>
+      <blockquote>Each test target has an alt attribute</blockquote>
+    </div>
+    <div class=example>
+      <p>**Example:** An HTML element with a duplicate ID does not meet the following expectation:</p>
+      <blockquote>No test target has the same ID</blockquote>
+    </div>
+    <div class=example>
+      <p>**Example:** An HTML `video` element without captions does not meet the following expectation:</p>
+      <blockquote>80% of all test targets has a caption</blockquote>
+    </div>
+    <div class=note>
+      <p>Expectations are true or false assertions about a test subject. Individual test targets can meet all expectations, even if some expectations are false about the test subject as a whole</p>
+    </div>
+  </dd>
+
   <dt><dfn>Outcome</dfn></dt>
   <dd>
     <p>One of three types of conclusions that come from evaluating an ACT Rule on a [=test subject=]. An outcome can be:<p>
     <ul>
       <li>**Inapplicable:** No part of the test subject matches the applicability</li>
-      <li>**Passed:** The test subject meets all expectations</li>
-      <li>**Failed:** The test subject does not meet all expectations</li>
+      <li>**Passed:** All expectations are true for the test subject</li>
```

Comment:
Shouldn't we se the "meets all expectations" definition here?

Reply:
I'm wondering why we are distinguishing "meets" and "is true".